src/app/frontend/common/components/labels/labels.scss

```diff
@@ -14,29 +14,7 @@
 
 @import '../../../variables';
 
-$label-margin: 0 $baseline-grid ($baseline-grid / 2) 0;
-
-// Style taken(mostly) from angular material md-chip style.
-.kd-labels {
-  background-color: $body;
-  border-radius: $baseline-grid;
-  display: inline-block;
-  margin: $label-margin;
-  max-width: 100%;
-}
-
-.kd-ellipsis-label {
-  display: block;
-  padding: ($baseline-grid / 4) $baseline-grid;
-}
-
 kd-labels {
```

Comment:
because this is the element name and not a class name that i could unify. perhaps I could add this to a class and apply class to both. Not sure if that is worth it though